### DISCUSSION TITLE
First time I ever saw this hack attempt

### DISCUSSION THREAD

**Discussion by 737simpilot** [CIDRAM/CIDRAM#470] [General]
> Report log.
> Date/Time: 2023-05-21T22:18:03-06:00
> IP address: 66.29.206.65
> Comments: Command injection detected in cookie.
> 
> Report log.
> Date/Time: 2023-05-21T22:18:06-06:00
> IP address: 66.29.206.65
> Comments: Command injection detected in cookie.
> 
> Report log.
> Date/Time: 2023-05-21T22:18:06-06:00
> IP address: 66.29.206.65
> Comments: Command injection detected in cookie.
> 
> Report log.
> Date/Time: 2023-05-21T22:18:06-06:00
> IP address: 66.29.206.65
> Comments: Command injection detected in cookie.

The IP was auto reported to AbuseIPDB (twice) and I was the only one to do so for this IP address. Further investigation reveals no open ports or other shenanigans that I can currently tell. Thinking about contacting the ISP's security department. In the some 9 years I've ran this website, I never saw this Sig ever get triggered. My host is auto aggregating the access logs so when that finishes I'll look there.

**Comment by Maikuolan** [comment]
Interesting to see this in action, and.. although it's never nice to see someone attempting to hack into our websites.. it is nice to see some validation for the cookie scanner module; some evidence that it's being useful. :-)

The signatures in question are rather tight, and not written in such a way that I would anticipate false positives, especially considering the vector in question (cookies, which generally shouldn't follow any pattern other than that defined by the specific software running at any given website), so.. unless a given website is setting something really funky, bizarre, or unusual for its cookies.. more than likely, it is exactly what it's being reported as (an attempt by the requesting entity to inject commands to the website software via cookies).

> Thinking about contacting the ISP's security department.

Might be a good idea. No idea whether anything'll come of it (really depends on the ISP in question, and how seriously they take these kinds of matters, which definitely varies from one provider to the next), but it shouldn't do any harm, at least. Of course, if you can pull anything useful from that log data after your host finishes aggregating it.. the more data and more evidence to go along with any potential future reports, the better.

> In the some 9 years I've ran this website, I never saw this Sig ever get triggered.

I've seen it triggered a few times before at a few websites that I help maintain where I have CIDRAM installed, but.. not often. Definitely a pretty rare event, I think.

**Comment by 737simpilot** [comment]
> Interesting to see this in action, and.. although it's never nice to see someone attempting to hack into our websites.. it is nice to see some validation for the cookie scanner module; some evidence that it's being useful. :-)

Exactly why I posted it here. Your script did the job again - in a real big way this time instead of at the IP level. 

>Might be a good idea. No idea whether anything'll come of it (really depends on the ISP in question, and how seriously they take these kinds of matters...


Yep, been down this road before and the last ISP I contacted wanted to see access logs which I didn't have at the time. This time I'll provide them.

**Comment by 737simpilot** [comment]
https://www.abuseipdb.com/check/66.29.206.65

There were four hits in total. The last three hit in quick succession at the same time. So two infractions were counted.
